src/resources/extensions/gsd/tests/custom-engine-loop-integration.test.ts

```
@@ -248,6 +248,7 @@ function makeMockDeps(overrides?: Partial<LoopDeps>): LoopDeps & { callLog: stri
     lifecycle: {
       enterMilestone: () => ({ ok: true, mode: "worktree", path: "/tmp/project" }),
     } as any,
```

Biome (2.4.14)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)